boefjes/boefjes/plugins/kat_cve_normalizer/normalize.py

```diff
@@ -5,34 +5,34 @@
 from octopoes.models import Reference
 from octopoes.models.ooi.findings import CVEFindingType, Finding, KATFindingType, RetireJSFindingType, SnykFindingType
 
+CVE_PATTERN = re.compile(r"CVE-\d{4}-\d{4,7}")
```

Comment:
The last number is specified as 4 or more digits, so more than 7 might also be possible in the future: https://cve.mitre.org/cve/identifiers/syntaxchange.html#new

```suggestion
CVE_PATTERN = re.compile(r"CVE-\d{4}-\d{4,}")
```